### PR TITLE
add copy button to landing page

### PR DESCRIPTION
**Pull request by NickyMeuleman**
First draft of adding copy button functionality to the home page.

I don't know the preferred methods of adding DOM-elements and styling them.
Right now, the PR uses a `<template>` tag and plain element css selectors, happy to change both approaches.

Opted to default to having a copy button there and removing it when the browser doesn't support the https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API, as most browsers support it https://caniuse.com/mdn-api_clipboard_writetext

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -44,10 +44,19 @@ <h1>Instant, themed Twitch chat overlays</h1>
 		allowtransparency="true"
 	>
 	</iframe>
+
+	<template id="output-template">
+		<p>Your URL is:</p>
+		<div>
+			<p class="url"></p>
+			<button name="Copy link to clipboard">Copy</button>
```
Comment (BenDMyers) [inline src/index.html:52]: Howdy! Can you help me understand what you're using the `name` attribute here for?

Comment (NickyMeuleman) [reply src/index.html:52]: I was trying to recreate the copy button found in codeblocks in the Gatsby docs site: https://www.gatsbyjs.com/docs/quick-start/#getting-started-with-gatsby

Comment (BenDMyers) [reply src/index.html:35]: Truthfully, I'm not sure that `name` is doing anything for them - unless they're using is as their query selector, a la `querySelector('[name="Copy link to clipboard"]')`, or in their styles.

Comment (NickyMeuleman) [reply src/index.html:35]: I'll remove that part. I assumed (wrongly) it served some accessibility function.

**Comment by NickyMeuleman** [comment]
Made some small changes.
Noticed a small change to the ESLint config as well, eventhough that's not directly related, all JS files in `scripts` are now considered to be ESM and all variables prefixed with `_` can remain unused.

**Review by BenDMyers — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**